spec/features/early_career_payments_claim_spec.rb

```diff
@@ -74,6 +74,11 @@
     # - In what academic year did you start your undergraduate ITT
     expect(page).to have_text(I18n.t("early_career_payments.questions.itt_academic_year.qualification.#{claim.eligibility.qualification}"))
     expect(page).to have_text("If you did a part time ITT")
+    expect(page).not_to have_text("2017 to 2018")
```

Comment:
According to the Capybara documentation you should use: `expect(page).to have_no_text("2017 to 2018")`